src/search_datastructures.hpp

```diff
@@ -408,11 +408,6 @@ class GlobalDataHolder
     using TQryIds       = TIds;
     using TSubjIds      = TIds;
 
-    /* indeces and their type */
-    using TDbIndex      = std::conditional_t<c_dbIndexType == DbIndexType::BI_FM_INDEX,
```

Comment:
Why don't we need this anymore? I thought we will include the bidirectional index again at some point.

Reply:
This alias is just a leftover of lambda2, it's independent of everything else (the index type is now part of the index file that we use as-is).